lib/api.js

```diff
@@ -4,6 +4,18 @@ import matter from 'gray-matter';
 
 const postsDirectory = join(process.cwd(), '_posts');
 
+export function getNextPostSlug(id) {
+  const nextSlug = `seminar-${id + 1}`;
+  if (fs.existsSync(join(postsDirectory, `${nextSlug}.md`))) return nextSlug;
+  return null;
+}
+
+export function getPreviousPostSlug(id) {
+  const prevSlug = `seminar-${id - 1}`;
+  if (fs.existsSync(join(postsDirectory, `${prevSlug}.md`))) return prevSlug;
+  return null;
+}
```

Comment:
How about: 

```suggestion
export function getPreviousPostSlug(id) {
  const prevSlug = `seminar-${id - 1}`;
  return fs.existsSync(join(postsDirectory, `${prevSlug}.md`)) ? prevSlug : null;
}
```

I don't like the null there, but it's explicit, so I'll just live with it.